server/routes/barcode/pdf/PdfController.test.ts

```diff
@@ -304,7 +304,6 @@ describe('PdfController', () => {
 
       try {
         await pdfController.downloadPdf(req as unknown as Request, res as unknown as Response)
-        fail('Was expecting pdfController.downloadPdf to have thrown an error but it did not')
```

Comment:
`fail` was an internal, undocumented jest function and should never have been used. It is not present in newer jest versions. All this seems to be used for is presenting a message.